File: backend/RiskManagementAPI/Models/Project.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RiskManagementAPI.Models
{
    [Table("project")]
    public class Project
    {
        [Column("id")] public int Id { get; set; }

        [Required]
        [StringLength(30, MinimumLength = 1)]
        [Column("name")]
        public string Name { get; set; }

        [Required] [Column("description")] public string Description { get; set; }
    }
}

File: frontend/riskManagementTool/src/app/model/project.ts
export class Project {

  public id: number;
  public name: string;
  public description: string;
  public riskRegisters: object;

  constructor(id?: number, name?: string, description?: string, riskRegisters?: object) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.riskRegisters = riskRegisters;
  }
}